distributionviewer/core/static/js/app/components/views/chart-list.js

```diff
@@ -9,7 +9,7 @@ export class ChartList extends React.Component {
       <section className="chart-list">
         {this.props.items.map(chart => {
           return (
-            <ChartContainer key={chart.name} isDetail={false} chartName={chart.name} />
+            <ChartContainer key={chart.name} isDetail={false} chartId={chart.id} chartName={chart.name} />
```

Comment:
`chartName` is used for the label of the chart (and was used as a class name in the DOM) so we still want it here. `key` is a react concept for iterators so that it has a unique identifier for each item (apparently this is part of how it diffs the DOM to figure out what to re-render) and it can be either a number or string as long as uniqueness is guaranteed.